src/LiveApi.js

```diff
@@ -13,6 +13,8 @@ let WebSocket = typeof window !== 'undefined' ? window.WebSocket : MockWebSocket
 const shouldIgnoreError = error =>
     error.message.includes('You are already subscribed to') ||
         error.message.includes('Input validation failed: forget');
+const shouldNotThrow = json => 
+    json.error.code === 'SelfExclusion' && json.msg_type === 'authorize';
 
```

Comment:
@qingweibinary shouldIgnore hide the error where 'shouldNotThrow' returns it to the front end. I wanted to refactor 'shouldIgnore' but is having difficulty demonstrating the issue and i identifying the right calls. Would do that any day i am able to demonstrate the issue.